src/data/evm_chains.json

```diff
@@ -598,6 +598,30 @@
       ],
       "image": "/images/chains/linea.svg"
     },
+    {
```

Comment:
Chains don't seem to be alphabetically sorted, but that's not an issue in this PR.
![image](https://github.com/axelarnetwork/axelar-docs/assets/3818101/4bcfa18e-7817-4500-9676-ed8875334c83)